server/sonar-server/src/main/java/org/sonar/server/platform/platformlevel/PlatformLevel1.java

```diff
@@ -69,8 +71,10 @@ public PlatformLevel1(Platform platform, Properties properties, Object... extraR
   public void configureLevel() {
     add(platform, properties);
     addExtraRootComponents();
+    Version apiVersion = ApiVersion.load(System2.INSTANCE);
     add(
-      SonarRuntimeFactory.create(System2.INSTANCE, SonarProduct.SONARQUBE, SonarQubeSide.SERVER),
+      new SonarQubeVersion(apiVersion),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SonarQubeVersion"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)